testingScenarios

```diff
@@ -165,3 +165,71 @@ SCENARIO("game of mixed spare after strike"){
         }
     }
 }
+
+SCENARIO("game have ten turns"){
+    GIVEN("counter of turns") {
+        int turnCounter = 0;
+        int maxTurns = 10;
+        playGame(turnCounter);
+        WHEN ("turnCounter == maxTurns") {
+            THEN ("Game end") {
+                REQUIRE_THAT(turnCounter, Equals(maxTurns));  
+            }
+        }
+    }
+}
+
+SCENARIO("each turn have maximum two tries"){
+    GIVEN("triesCounter and turnCounter") {
+        int turnCounter = 1;
+        int triesCounter = 0;
+        int maxTries = 2;
+        playGame(turnCounter);
+        WHEN ("triesCounter == maxTries") {
+            THEN ("Increment turnCounter") {
+                int expectedValue = 2;
+                REQUIRE_THAT(turnCounter, Equals(expectedValue));  
+            }
+        }
+    }
+}
+
+SCENARIO("first ball in a turn knocks down all ten pins"){
+    GIVEN("turnCounter") {
+        int turnCounter = 1;
+        WHEN ("ball knocks down all pins") {
+                turnCounter++;
+            THEN ("Increment turnCounter") {
+                int expectedValue = 2;
+                REQUIRE_THAT(turnCounter, Equals(expectedValue));  
+            }
+        }
+    }
+}
+
+
+SCENARIO("first ball in a turn knocks down all ten pins"){
+    GIVEN("strikeScore and score of next two balls") {
+        std::vector<std::pair<int, int>> score { {10, 0}, {6, 2}, {3, 6} };
+        WHEN ("ball knocks down all pins") {
+                int totalScore = countScore(score);
+            THEN ("totalScore should equal to expectedValue") 
+                int expectedValue = 18;
```

Comment:
Does expected points shouldn't be 35? (10 + (6+2)*2 + (3 + 6))

Reply:
I think no because if you get the Strike you have 10 points and sum of points at next two throws not rounds :)

Reply:
it depend on how will countScore(std::vector) work.
But i think it will be counting whole score vector which we will send to the method. In that case 35.